mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -66,58 +61,42 @@ public static String normalizeUrl(String sourceUrl) {
         return normalizedUrl;
     }
 
-    public static List<OpenHABSitemap> parseSitemapList(Document document) {
-        List<OpenHABSitemap> sitemapList = new ArrayList<OpenHABSitemap>();
-        NodeList sitemapNodes = document.getElementsByTagName("sitemap");
-        if (sitemapNodes.getLength() > 0) {
-            for (int i = 0; i < sitemapNodes.getLength(); i++) {
-                Node sitemapNode = sitemapNodes.item(i);
-                OpenHABSitemap openhabSitemap = new OpenHAB1Sitemap(sitemapNode);
-                sitemapList.add(openhabSitemap);
-            }
+    public static List<? extends Sitemap> parseSitemapList(String json) {
+        ObjectMapper mapper = new ObjectMapper();
+        if (json.startsWith("<?xml")) {
```

Comment:
If "json" can be a xml, maybe json is a misleading name?

Reply:
Fixed! :)